scripts/gen_github_action_config/main.go

```diff
@@ -0,0 +1,231 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"log"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/shurcooL/githubv4"
+	"golang.org/x/oauth2"
+)
+
+func main() {
+	if err := generate(context.Background()); err != nil {
+		log.Fatal(err)
+	}
+}
+
+func generate(ctx context.Context) error {
+	allReleases, err := fetchAllReleases(ctx)
+	if err != nil {
+		return fmt.Errorf("failed to fetch all releases: %w", err)
+	}
+
+	cfg, err := buildConfig(allReleases)
+	if err != nil {
+		return fmt.Errorf("failed to build config: %w", err)
+	}
+
+	if len(os.Args) != 2 { //nolint:gomnd
+		return fmt.Errorf("usage: go run .../main.go out-path.json")
+	}
+	outFile, err := os.Create(os.Args[1])
+	if err != nil {
+		return fmt.Errorf("failed to create output config file: %w", err)
+	}
+	defer outFile.Close()
+	enc := json.NewEncoder(outFile)
+	enc.SetIndent("", "  ")
+	if err = enc.Encode(cfg); err != nil {
+		return fmt.Errorf("failed to json encode config: %w", err)
+	}
+
+	return nil
+}
+
+type logInfo struct {
+	Warning string `json:",omitempty"`
+	Info    string `json:"omitempty"`
```

Comment:
Seems like a typo, probably `,omitempty` is expected here

Reply:
you are right, thank you, we need a linter for that :)